agcaballero/portfolio/src/main/webapp/script.js

```diff
@@ -11,3 +11,46 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
+/* Returns the link to a random post on the blog */
```

Comment:
JSDoc style prescribes `/** Returns the link to a random post on the blog */` (slash-star-star ...)
https://google.github.io/styleguide/jsguide.html#jsdoc-general-form

This comment also applies to functions below.

Reply:
Done